qa/L0_nullchar_string/nullchar_string_client.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
```

Comment:
"#"

Reply:
Just delete the blank line to match standard style

Reply:
Done